bibtex-actions.el

```diff
@@ -266,8 +274,8 @@ TEMPLATE."
             field-value
             (if (> field-width 0)
                 field-width
-              (- width (cddr format)))
-            0 ?\s)))))))
+              (- final-width width)
```

Comment:
This seems to be the line that causes the problem, but this is not the solution.